docs/testing.rst

```diff
@@ -660,11 +660,11 @@ for accessing to the server.
 
    .. attribute:: handler
 
-      :class:`aiohttp.web.WebServer` used for HTTP requests serving.
+      :class:`aiohttp.web.Server` used for HTTP requests serving.
 
    .. attribute:: server
 
-      :class:`asyncio.AbstractServer` used for managing accepted connections.
```

Comment:
Importing `asyncio.AbstractServer` works. It's just not documented. So the reference was correct, it just needs to be ignored.